spec/helpers/orphan_helper_spec.rb

```diff
@@ -1,20 +1,36 @@
 require "rails_helper"
 
 RSpec.describe OrphanHelper, type: :helper do
-  describe "#child_or_adult?" do
-    context "when orphan is a child" do
-      it "returns 'child'" do
-        orphan = double "orphan", age_in_years: 17
+  describe "#active_adult?" do
+    context "when active child" do
+      example do
+        orphan = double "orphan", adult?: false, active?: true
 
-        expect(child_or_adult? orphan).to eq "child"
+        expect(active_adult?(orphan)).to be_nil
       end
     end
 
-    context "when orphan is an adult" do
-      it "returns 'adult'" do
-        orphan = double "orphan", age_in_years: 18
+    context "when inactive child" do
+      example do
+        orphan = double "orphan", adult?: false, active?: false
 
-        expect(child_or_adult? orphan).to eq "adult"
+        expect(active_adult?(orphan)).to be_nil
+      end
+    end
+
+    context "when inactive adult" do
+      example do
+        orphan = double "orphan", adult?: true, active?: false
+
+        expect(active_adult?(orphan)).to be_nil
```

Comment:
Agreed that the method name was poorly chosen. Renaming it to `orphan_highlight_class` with no other changes works the best for me. In this case, I think testing for a `nil` return is the right thing to do as we want to make sure that the method does not return any CSS class unless the conditions are met.